src/test/java/org/heigit/ohsome/ohsomeapi/controller/TestProperties.java

```diff
@@ -10,4 +10,5 @@
   public static final String JUNIT = System.getProperty("junit");
   public static final String INTEGRATION = System.getProperty("integration");
   public static final String SERVER = "http://localhost:";
+  public static final double DELTA_PERCENTAGE = 0.5 / 100; // 0.5 %
```

Comment:
I could not find a specific line regarding putting comments in-line vs. before the line in the Google Java style guide, but in the ohsome API it's normally written before the respective line. I found one exception for that though [here](https://github.com/GIScience/ohsome-api/blob/c9da9c78fa5834c9b530211697fbebba7a3ae93f/src/main/lombok/org/heigit/ohsome/ohsomeapi/inputprocessing/InputProcessor.java#L129), but I'd prefer to keep it to that one exception :)

If you really want to write a comment here, I'd suggest to change it to something like:
```suggestion
  // gives a percentage of 0.5%
  public static final double DELTA_PERCENTAGE = 0.5 / 100;
```

Reply:
I think it is easy enough to understand that dividing by 100 returns a percentage. I will just remove this comment from the PR.

Reply:
I agree, it should also be obvious from the variable name and the given value. But let @bonaparten due this change as it was also him who added this part, or at least he should give his opinion on that as well.

Reply:
No problem with deleting the comment.